Makefile.gh

```diff
@@ -69,6 +69,9 @@ build-wheel: pip
 			cd -;\
 		fi;\
 	done
+	ls
```

Comment:
I understand these were used for debugging, but this is just a reminder to remove those two lines.